packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -58,53 +58,24 @@ class SlidePanel extends React.Component {
     super(props);
     this.setPanelNode = this.setPanelNode.bind(this);
     this.mainNode = React.createRef();
-    this.handleTransitionEnd = this.handleTransitionEnd.bind(this);
-    this.preparePanelForTransition = this.preparePanelForTransition.bind(this);
 
     this.isHidden = !props.isOpen;
```

Comment:
From what I'm seeing, I don't think we need to manage this internal property anymore?

Reply:
without animations, isHidden is just !isOpen all the time.

Reply:
https://github.com/cerner/terra-framework/pull/617/commits/f8bc8503300c1f9a66e7a18aa33b80ed7cf64d3b

Reply:
wait a sec....do we even need to toggle `aria-hidden` inside componentUpdate if we use `isOpen` inside the render function?

Reply:
I don't think so. Remove all the things.